pkg/kubelet/kubelet.go

```diff
@@ -1773,6 +1772,8 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 
 		switch u.Op {
 		case kubetypes.ADD:
+			// Perform limits check for resolv.conf
+			kl.checkLimitsForResolvConf()
```

Comment:
What I meant by checking it was that `formDNSSearchFitsLimits` already checks the limits and truncates if needed.  We don't need ANOTHER function that just opens, parses, verifies, logs, and returns, do we?  Or am I missing some reason to do both?